Server/src/BaseQuasarServer.cpp

```diff
@@ -284,9 +359,18 @@ int BaseQuasarServer::initializeEnvironment()
     const int ret = 0;
 #endif    
     initializeLogIt();
+    logEnvironment();
```

Comment:
That's what I meant above -- why we couldn't just print command line args directly after initializing LogIt? Why to store it?

Reply:
my response in comment above. SO wil close this one (leabing single place for comments)